sklearn/linear_model/_ridge.py

```diff
@@ -2272,7 +2281,7 @@ class RidgeCV(
         Alpha corresponds to ``1 / (2C)`` in other linear models such as
         :class:`~sklearn.linear_model.LogisticRegression` or
         :class:`~sklearn.svm.LinearSVC`.
-        If using Leave-One-Out cross-validation, alphas must be positive.
+        If using Leave-One-Out cross-validation, alphas must be > 0.
```

Comment:
```suggestion
        If using Leave-One-Out cross-validation, alphas must be strictly positive.
```
or
```suggestion
        If using Leave-One-Out cross-validation, `alphas > 0` is required.
```
I find the bare > not optimal to read - personal taste. Same in `RidgeClassifierCV`.

Reply:
Thanks, I was hesitant about that too. I've amended to 'strictly positive'.